docs/pages/versions/unversioned/sdk/print.mdx

```diff
@@ -158,6 +158,8 @@ const { uri } = await Print.printToFileAsync({
 });
 ```
 
+If `useMarkupFormatter` is set to `true`, setting margins may cause a blank page to appear at the end of your printout. To prevent this, make sure your HTML string is a well-formed document, including `<!DOCTYPE html>` at the top of the string.
```

Comment:
```suggestion
If `useMarkupFormatter` is set to `true`, setting margins may cause a blank page to appear at the end of your printout. To prevent this, make sure your HTML string is a well-formed document, including `<!DOCTYPE html>` at the beginning of the string.
```